[7.14] docs: What’s new? (#939)
elastic/observability-docs: docs/en/observability/whats-new.asciidoc

Here are the highlights of what's new and improved in {minor-version}.

Other versions:
{observability-guide-all}/7.13/whats-new.html[7.13] |
{observability-guide-all}/7.12/whats-new.html[7.12] |
{observability-guide-all}/7.11/whats-new.html[7.11] |
{observability-guide-all}/7.10/whats-new.html[7.10] |
https://www.elastic.co/blog/whats-new-elastic-observability-7-9-0-unified-agent-kpi-overview-dashboard[7.9] |
https://www.elastic.co/blog/elastic-observability-7-8-0-released[7.8] |
https://www.elastic.co/blog/elastic-observability-7-7-0-released[7.7] |
https://www.elastic.co/blog/elastic-observability-7-6-0-released[7.6] |
https://www.elastic.co/blog/elastic-observability-7-5-0-released[7.5] |
https://www.elastic.co/blog/elastic-observability-update-7-4-0[7.4]

[discrete]
== Give {minor-version} a try

Try {minor-version} now by deploying {es} and {kib} on
{ess-product}[Elastic Cloud] or
by https://www.elastic.co/start[downloading them].

// tag::whats-new[]
[discrete]
== {agent}, {fleet}, and Integrations now generally available (GA)

To make it easier to integrate your systems and applications with the {stack}, we are excited to announce

> a major improvement in the 7.14 release.

Reviewer: This is weird to me. We say we're announcing **a major improvement**, and then discuss three different enhancements. I think we should pluralize this. Something like this:

```suggestion
three significant improvements in the 7.14 release.
```

First, we are launching the generally available (GA) release of our {fleet-guide}/elastic-agent-installation-configuration.html[{agent}],
a single, unified agent for Observability and Security. With fewer things to configure and install,
a single agent will simplify data onboarding.

Second, we are launching the GA release of {fleet-guide}/fleet-overview.html[{fleet}], which lets you centrally manage an entire fleet
of {agent}s at scale. It acts as a control plane that gives you a real-time view of the agent status. In addition, it allows

> you to upgrade {agent}s remotely, execute queries on each host, and also contains security threats.

Reviewer: ```suggestion
you to upgrade {agent}s remotely, execute queries on each host, and also contain security threats.
```

OK so this is actually a really funny typo because `and also contain security threats` means something very different from `and also contains security threats` 😂

Author: Oops! Great spot. Thanks.

[role="screenshot"]
image::images/7.14-fleet.png[{fleet}]

Third, we are launching our new {integrations} app in {kib} along with our first set of GA {integrations}. With a single click, you can
integrate your systems and applications with the {stack}.

> {integrations} provide out of the box log parsing, dashboards, machine learning jobs and more. In addition, you can deploy

Reviewer: ```suggestion
{integrations} provide out of the box log parsing, dashboards, machine learning jobs, and more. In addition, you can deploy
```

> {integrations} provide out of the box log parsing, dashboards, machine learning jobs, and more. In addition, you can deploy

Reviewer: I can never remember which way our style guide goes on this one

```suggestion
{integrations} provide out-of-the-box log parsing, dashboards, machine learning jobs, and more. In addition, you can deploy
```

[role="screenshot"]
image::images/7.14-agent-unenroll-timeout.png[Unenrollment timeout setting in {fleet}]

[discrete]
== Removing Beats central management

Beats Central Management was an old solution for centrally managing Beats. It was released as a beta and was deprecated several releases ago.
We are now removing it from {kib}. We recommend that you switch to our new GA release of {fleet} instead.

[discrete]
== Advanced metrics collection from Couchbase Sync Gateway

For 7.14, we’re enhancing the Couchbase integration with support for an extended list of metrics. You can now use the
{metricbeat-ref}/metricbeat-module-couchbase.html[Couchbase module] in {metricbeat} to collect server metrics from
https://docs.couchbase.com/sync-gateway/current/stats-monitoring.html[Sync Gateway].

The module uses the {metricbeat} Couchbase input to get additional metrics related to resources, cache, database, delta sync, doc imports,
CBL replication, security, and GSI views. With these expanded metrics, we have made sure that only the essential metrics are collected to keep
the consumption of resources appropriate.

[discrete]
== Enhanced granularity of the AWS integration

> The pay-as-you-go model gets applied to each granular component of any cloud IaaS PaaS to create more advanced cloud offerings.

Reviewer: Should these be differentiated?

```suggestion
The pay-as-you-go model gets applied to each granular component of any cloud IaaS or PaaS to create more advanced cloud offerings.
```

Author: Yeah they should be. Thanks.
